### PR TITLE
Write up how push changes to a private GH repo from a SRC server

### LINKED ISSUE #104: Document how pull to private repo or push to any repo
Write for hydrologists first time users

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes #104

### CODE REVIEW

**Comment by sverhoeven** [comment]
@RolfHut are instructions at https://github.com/eWaterCycle/infra/blob/104-doc-notebook2github/USER.md#save-notebooks-to-github what you wanted?

File: README.md [hunk 1]
```diff
@@ -109,10 +109,9 @@ For a new CO make sure
 * application is allowed to be used by CO.
```
Comment (Peter9192) [inline README.md:109]: Note DOI badge doesn't work

File: README.md [hunk 2]
```diff
@@ -109,10 +109,9 @@ For a new CO make sure
...
 * data storage item and home dir are created for the CO
 
-End user should be invited to CO and then
+End user should be invited to CO so they can login.
 
-1. to login to Jupyter, he/she should setup TOTP on [SRC dashboard profile page](https://sbs.sram.surf.nl/profile)
-2. optionally to ssh into machine, the ssh pub key must be added to [https://sbs.sram.surf.nl/profile](https://sbs.sram.surf.nl/profile)
```
Comment (Peter9192) [inline README.md:115]: Maybe link to user documentation from README.md?

**Review by Peter9192 — APPROVED** [review]

**Comment by RolfHut** [comment]
yes, very much so!
-- 
PS my working hours may not be your working hours. Please only reply in
your working hours

*--*
*dr. ir. Rolf Hut*
MacGyver Associate Professor in Water Resources and Maker Education at Delft
University of Technology
<http://www.tudelft.nl/>
Public speaker on Technology and Innovation for Everyone
Columnist at De Ingenieur <https://www.deingenieur.nl/artikel/quarantandem>
Writer of Rolfs Maakbare Wereld
<http://rolfhut.nl/rolfsmaakbarewereldbolcom> and Knotsgekke Surprises voor
Iedereen <http://rolfhut.nl/surpriseboek>.
www.rolfhut.nl/english
@RolfHut ***@***.***>

*Recent papers*
Hut et al.: Easy to build low-power GPS drifters with local storage and a
cellular modem made from off-the-shelf components
<https://doi.org/10.5194/gi-9-435-2020>, Geoscientific Instrumentation,
Methods and Data Systems 2020
Hut et al.: Teaching a hands-on course during corona lockdown: from
problems to opportunities <https://doi.org/10.1088/1361-6552/abb06a>,
Physics Education 2020
Hut et al.: Taking a Breath of the Wild: are geoscientists more effective
than non-geoscientists in determining whether video game world landscapes
are realistic? <https://doi.org/10.5194/gc-2-117-2019>, Geoscience
Communication 2019

File: README.md (hunk 1 above)
Comment (sverhoeven) [reply README.md:109]: Works for me: image is blue and links to Zenodo page with latest version.

**Comment by sverhoeven** [comment]
Thanks for reviewing. I added a link to the user guide at the top of the README as well.

**Pull request merged**